policies/cookie-policy.md

```diff
@@ -0,0 +1,74 @@
+- Identifying and assessing the capability of the User’s device to deliver various content depending on the browser device being used.
+- Improve security by recognizing suspicious activity or repeated failed login attempts.
+- Analyze aggregated data to optimize user experience and site performance.
+
+## What Cookies Do We Use?
+
+**Essential cookies** - We may store cookies on your device if necessary to enable us to offer and provide Services, including intended user functionality and use of the Website. They are also required for you to navigate through the Website and benefit from all the available features. These cookies cannot be switched off and are set for fundamental options on the Website, such as logging in and forms, as well as set preferences, such as timezone or languages (as noted above).
+
+**Analytical cookies** - These cookies help us learn how you interact with the Website and its features, which parts you use the most, and how you otherwise engage with the Services. This helps us track various usage statistics and better understand how much time you spend on the Website and otherwise interacting with the Services. These statistics are intended to help us to improve the Website and the Services related thereto.
+
+**Functional Cookies** - These cookies help us provide you with better functionality of the Website and otherwise in connection with the Services, including a more personalized experience. Third-party providers can also place these cookies you expressly consent to when you accept this Cookie Policy and the Privacy Policy by accessing the Website and/or otherwise using the Services provided in connection therewith. You may block these cookies, but if you do, parts of the Website do not function the way we intended.
+
+## How to Block Cookies
+
+You may stop or restrict the placement of cookies and other tracking technologies, such as pixel tags. You can do this by blocking them on your device or by removing them and adjusting preferences on your browser. This may vary from device to device and from browser to browser.
+
+Please see the links below for guidance on how to modify your web browser’s settings on some popular browsers:
+
+[Microsoft Edge](https://allaboutcookies.org/how-to-clear-cookies-microsoft-edge){target=\_blank}
+[Internet Explorer](https://allaboutcookies.org/how-to-clear-cookies-microsoft-edge){target=\_blank}
+[Mozilla Firefox](https://allaboutcookies.org/how-to-clear-cookies-firefox){target=\_blank}
+[Google Chrome](https://allaboutcookies.org/how-to-clear-cookies-chrome){target=\_blank}
+[Apple Safari](https://allaboutcookies.org/how-to-clear-cookies-safari){target=\_blank}
+
+Please note that once you correctly block cookies on your browser, parts of the Website may become unavailable or otherwise not function properly. Refer to your specific device's user guide for detailed instructions on managing cookies for mobile devices.
+
+## Cookie List
+
+Essential Cookies:
+
+
+Analytical Cookies:
+
```

Comment:
@albertov19 tagging you here so we don't forget about this

Reply:
@eshaben let's wait for the soft launch to confirm the cookies and then we can add them here :)